Algorithm.CSharp/LimitIfTouchedRegressionAlgorithm.cs

```diff
@@ -0,0 +1,123 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+using QuantConnect.Data;
+using QuantConnect.Interfaces;
+using QuantConnect.Orders;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// Basic algorithm demonstrating how to place LimitIfTouched orders.
+    /// </summary>
+    /// <meta name="tag" content="trading and orders" />
+    /// <meta name="tag" content="placing orders" />`
+    /// <meta name="tag" content="limit if touched order"/>
+    public class LimitIfTouchedRegressionAlgorithm: QCAlgorithm, IRegressionAlgorithmDefinition
+    {
+        private SubmitOrderRequest request;
+
+        /// <summary>
+        /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
+        /// </summary>
+        public override void Initialize()
+        {
+            SetStartDate(2013, 10, 07);  //Set Start Date
+            SetEndDate(2013, 10, 15);    //Set End Date
+            SetCash(100000);             //Set Strategy Cash
+            // Find more symbols here: http://quantconnect.com/data
+            AddSecurity(SecurityType.Equity, "SPY", Resolution.Second);
+        }
+        
+        /// <summary>
+        /// OnData event is the primary entry point for your algorithm. Each new data point will be pumped in here.
+        /// </summary>
+        /// <param name="data">TradeBars IDictionary object with your stock data</param>
+        public override void OnData(Slice data)
+        {
+            if (data.ContainsKey("SPY"))
+            {
+                if (Time.Second == 0 && Time.Minute == 0 && Transactions.GetOpenOrders().Count < 1)
+                {
+                    var goLong = Time < StartDate.AddDays(2);
+                    var negative = goLong ? 1 : -1;
+                    request = new SubmitOrderRequest(OrderType.LimitIfTouched, SecurityType.Equity, "SPY",
+                        negative * 10, 0, data["SPY"].Price - (decimal) 0.5 * negative,
+                        data["SPY"].Price - (decimal) negative, UtcTime, $"LIT - {UtcTime}");
+                    Transactions.AddOrder(request);
```

Comment:
Curious why this approach versus calling `LimitIfTouchedOrder()`? Let's please add an order update of the LIT order asserting it worked, simple just the quantity

Reply:
Was to display that order-interactions were possible but agree it would make more sense to utilize it by changing the order quantity!